lib/view_component/base.rb

```diff
@@ -65,12 +65,14 @@ def self._deprecated_generate_mattr_accessor(name)
     # Returns HTML that has been escaped by the respective template handler.
     #
     # @return [String]
-    def render_in(view_context, &block)
+    def render_in(view_context, global_buffer_in_use = false, &block)
```

Comment:
```suggestion
    def render_in(view_context, global_buffer_in_use = false, &block)
```

I'm a little hesitant about modifying this method since it's implementing a hook from Rails. Thoughts on setting some variable on `ViewComponent::Base` instead? e.g. `ViewComponent.global_buffer_in_use?`?

Reply:
I think it should be ok to use an instance variable here.